Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ReflectometryILLPreprocess.py

```diff
@@ -10,38 +10,38 @@
 
 from mantid.api import (AlgorithmFactory, DataProcessorAlgorithm, FileAction, ITableWorkspaceProperty,
                         MatrixWorkspaceProperty, MultipleFileProperty, PropertyMode, WorkspaceUnitValidator)
-from mantid.kernel import (CompositeValidator, Direction, FloatArrayBoundedValidator, FloatArrayLengthValidator,
+from mantid.kernel import (CompositeValidator, Direction,
                            IntArrayLengthValidator, IntArrayBoundedValidator, IntArrayProperty,
                            IntBoundedValidator, Property, StringListValidator)
 from mantid.simpleapi import (AddSampleLog, CalculatePolynomialBackground, CloneWorkspace, ConvertUnits,
-                              CreateEmptyTableWorkspace, Divide, ExtractMonitors, Fit,
-                              Integration, LoadILLReflectometry, MergeRuns, Minus, mtd, NormaliseToMonitor,
-                              RebinToWorkspace, Scale, Transpose)
+                              Divide, ExtractMonitors, FindReflectometryLines, LoadAndMerge, Minus, mtd,
+                              NormaliseToMonitor, RebinToWorkspace, Scale, SpecularReflectionPositionCorrect, Transpose)
 import numpy
-import os.path
 import ReflectometryILL_common as common
 
 
 class Prop:
     BEAM_ANGLE = 'BraggAngle'
     BEAM_CENTRE = 'BeamCentre'
-    BEAM_POS_WS = 'BeamPositionWorkspace'
+    BEAM_POS_WS = 'DirectBeamPosition'
```

Comment:
I think we should have a `DirectLineWorkspace` for the direct beam instead which will be used for the reflected angle calibration.

Reply:
ok